client/components/Trends/TrendsVisualization.vue

```diff
@@ -89,8 +99,9 @@ export default {
 };
 </script>
 <style>
+
 canvas {
-  width: 50% !important;
-  height: 70% !important;
+  width: 30rem !important;
+  height: 15rem !important;
```

Comment:
I wonder if given our demographic, it might be good to make the graphs larger. Perhaps 60rem and 30rem? And it would be nice to make the font larger for graph titles and labels as well :)